dashboard/app/models/regional_partner.rb

```diff
@@ -59,6 +57,14 @@ def program_manager=(program_manager_id)
     )
   end
 
+  def contact
+    User.find(read_attribute(contact_id)) || program_managers.first
```

Comment:
[find](http://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-find) will raise `RecordNotFound` if the id isn't found, which will always be the case for `nil`.

Please add tests (which would have caught this) for both cases: the contact_id and program manager fall through.